rotkehlchen/tests/unit/decoders/test_aave_v3.py

```diff
@@ -1219,40 +1219,40 @@ def test_aave_v3_withdraw_eth(scroll_inquirer, scroll_accounts) -> None:
             sequence_index=63,
             timestamp=timestamp,
             location=Location.SCROLL,
-            event_type=HistoryEventType.SPEND,
-            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            event_type=HistoryEventType.RECEIVE,
```

Comment:
interest always comes at the end. So this will require changes in the decoder to make it happen